packages/survey-creator-knockout/src/components/question.html

```diff
@@ -12,22 +13,28 @@
         data-bind="event: {pointerdown: (model, event)=>{startDragSurveyElement(event)}}"
       ></div>
     </div>
+    <!-- ko template: { name: 'svc-question-adoner-content', data: $data } --><!-- /ko -->
 
-    <!-- ko if: isEmptyElement -->
-    <div class="svc-panel__placeholder_frame">
-      <div
-        class="svc-panel__placeholder"
-        data-bind="text: placeholderText"
-      ></div>
-    </div>
-    <!-- /ko -->
-
-    <!-- ko component: { name: 'sv-template-renderer', params: { componentData: null, templateData: templateData } } -->
+    <!-- ko if: adornerComponent -->
+    <!-- ko component: { name: adornerComponent, params: { model: $data } } --><!-- /ko -->
     <!-- /ko -->
 
     <div class="svc-question__content-actions">
       <!-- ko component: { name: 'sv-action-bar', params: { model: actionContainer } } -->
       <!-- /ko -->
     </div>
   </div>
+  <!-- /ko -->
+  <!-- ko ifnot: question.isInteractiveDesignElement -->
+  <!-- ko template: { name: 'svc-question-adoner-content', data: $data } --><!-- /ko -->
+  <!-- /ko -->
 </div>
+<script type="text/html" id="svc-question-adoner-content">
```

Comment:
Unless I'm mistaken, this leads to the duplicated element id error due to this template will be rendered for each question. Right now we don't (shouldn't) use templates in Creator V2 and will get rid of them in survey-knockout (it will be the right way)

Reply:
@tsv2013 OK. I have removed the template. There is a small code duplication in question.html file now. I use components in other places.

Thank you,
Andrew